universal-application-tool-0.0.1/app/views/components/Modal.java

```diff
@@ -112,4 +121,20 @@ private void setOptionalFields() {
       triggerButtonText = Optional.ofNullable(triggerButtonText).orElse(modalTitle);
     }
   }
+
+  public enum Width {
```

Comment:
suggestion: Given this, should we use one of these as the "default" width for modals?  And/or add AUTO to this?

Reply:
done!